overops/values.yaml

```diff
@@ -20,6 +20,9 @@ overops-collector:
 overops-event-generator:
   enabled: true
```

Comment:
We can set this to false as part of this PR.

Reply:
Done